clouds/clouds-azure/src/test/java/azure/armTemplates/archiveDeploy/vm/suitetests/AzureVmUndeployedSecondTest.java

```diff
@@ -0,0 +1,40 @@
+package azure.armTemplates.archiveDeploy.vm.suitetests;
+
+
+import azure.core.identification.AzureVirtualMachine;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import sunstone.api.Parameter;
+import sunstone.api.WithAzureArmTemplate;
+import sunstone.api.inject.Hostname;
+
+import java.io.IOException;
+
+import static azure.armTemplates.AzureTestConstants.IMAGE_REF;
+import static azure.armTemplates.AzureTestConstants.instanceName;
+import static azure.armTemplates.archiveDeploy.vm.VmDeploySuiteTests.vmDeployGroup;
+
+@WithAzureArmTemplate(parameters = {
+        @Parameter(k = "virtualMachineName", v = instanceName),
+        @Parameter(k = "imageRefId", v = IMAGE_REF)
+},
+        template = "azure/armTemplates/eap.json", group = vmDeployGroup, perSuite = true)
+public class AzureVmUndeployedSecondTest {
+    @AzureVirtualMachine(name = instanceName, group = vmDeployGroup)
+    Hostname hostname;
+
+    @Test
+    public void test() throws IOException {
+        OkHttpClient client = new OkHttpClient();
+
+        Request request = new Request.Builder()
+                .url("http://" + hostname.get() + ":8080/testapp")
```

Comment:
So we rely here on fact app `testapp` was deployed in previous test???

Reply:
We rely on the fact the Suite runs test classes sequentially. The test is supposed to be run after `AzureWebAppDeployFirstTest` and verify the artifact was undeployed correctly.

Reply:
I have added a javadoc.

Reply:
ok